server/api/crud/model_monitoring/deployment.py

```diff
@@ -1198,11 +1209,11 @@ def set_credentials(
             )
         )
 
-        self._create_stream_output(
-            stream_path=secrets_dict.get(
+        if not mlrun.mlconf.is_ce_mode():
+            stream_path = secrets_dict.get(
                 mlrun.common.schemas.model_monitoring.ProjectSecretKeys.STREAM_PATH
             )
-        )
+            self._verify_v3io_access(stream_path)
```

Comment:
What about Kafka?
Can you create `_verify_stream_access` and to verify it according to the stream path.

Reply:
Since it's not a regression in this PR, and not a part of the requirement, this will need a separate ticket/PR.

Reply:
Opened [ML-8123](https://iguazio.atlassian.net/browse/ML-8123) to track this.